include/crow/socket_adaptors.h

```diff
@@ -38,11 +38,11 @@ namespace crow
     struct SocketAdaptor
     {
         using context = void;
-        SocketAdaptor(asio::io_service& io_service, context*):
-          socket_(io_service)
+        SocketAdaptor(asio::io_context& io_context, context*):
+          socket_(io_context)
         {}
 
-        asio::io_service& get_io_service()
+        asio::io_context& get_io_service()
```

Comment:
Shouldn't that also be renamed to get_io_context() to be consistent?

Reply:
Missed that, this is fixed now